### ISSUE TITLE
Reason about the calander conversation algorithm

### ISSUE THREAD

**Issue by m3hari** [Miqe/ken.js#16]
Currently, the conversation algorithm seems to lack a research(mathematical)  background.  It just follows some kind of pattern.
I think the conversation algorithm need to be based on some decent research background.

The following links might be a reference
- Beyene-Kudlek algorithm http://www.geez.org/Calendars/
- http://etd.aau.edu.et/bitstream/123456789/9283/1/Zenebe%20Nigussie.pdf

**Comment by miqe** [comment]
True, i came up with the patter with some research based on printed previous year calendar which are distributed by multiple companies. From my perspective its not the pattern/algorithm which is incorrect but the implementation. With the help of automated testing, which is on the previous commit https://github.com/Miqe/ken.js/commit/9f253272ec46885203f3826b366ae8a28d54c9b5 the pattern/algorithm implementation can be perfected thus leading to **a new way** of converting Ethiopian date to Gregorian and vice versa.

**Comment by m3hari** [comment]
Great! It would be awosem if the `alogithm/pattern` can be clearly included in the README section.
